### ISSUE TITLE
Reduce memory usage

### ISSUE THREAD

**Issue by stheophil** [stheophil/wahlversprechen#18]
Heroku complains that the site exceeds the 512M memory limit. Memory usage increases permanently. Maybe the site cache counts against the memory limit, maybe its a memory leak.

**Comment by stheophil** [comment]
The site cache is kept in memory because there is no persistent storage. 

The cache used to parse the feeds seems to dominate.

**Comment by optikfluffel** [comment]
Heroku offers a couple of [Add-Ons](https://addons.heroku.com) for caching. Maybe a Redis or Memcached could help.

**Comment by stheophil** [comment]
Of course I was trying to keep Heroku free :-) 

The problem was that I couldnt reproduce the behavior locally. Even with the python script that emulates users of the site permanently. 

The app continually used relatively low amounts of mem and was easily constrained to 512M. I have no idea what counted against the Heroku mem limit. I've moved the server off of Heroku anyway, they had a bad bug that caused days of downtime.

 It's hosted by codefor.de now. The issue is still open until I've managed to check if the behavior is OK now.
